webviz_core_components/wrapped_components/labeled_container.py

```diff
@@ -0,0 +1,32 @@
+from typing import Any
+
+from dash import html
+
+
+class LabeledContainer(html.Div):
+    """Creates a container with a header label and padded content
+
+    Keyword arguments:
+
+    - children (a list of or a singular dash component, string or number; required):
+        The children of this component.
+
+    - label (str; optional):
+        The text of the label
+
+    - className (string; optional):
+        Additional css class to apply
+
+    """
+
+    def __init__(
+        self, children: Any, label: str = "", className: str = "", **kwargs: Any
+    ) -> None:
+        super().__init__()
+        self.className = className + "webviz-labeled-container"
```

Comment:
Class name see above. Moreover, if a class name is given, wouldn't the resulting `self.className` become `givenClassNamewebviz-labeled-container`?

Reply:
True - fixed